generators/authentication/templates/config/passport.js

```diff
@@ -151,6 +151,7 @@ module.exports = {
      */
     onPassportAuth(req, res, error, user, info) {
       if (error || !user) return res.negotiate(error || info);
+      if (info.name === 'TokenExpiredError') info.status = 401; return res.negotiate(info)
```

Comment:
Why can't you leave updating `info.status` ? What `info` contains in case of `TokenExpiredError`? I was hoping to write literally `if (info.name === 'TokenExpiredError') return res.negotiate(info)`. If you write without updating `info.status = 401` what causes it has ?